### PR TITLE
Running single container for nodes containers

### PR DESCRIPTION
**Pull request by Yarboa**
Resolve images issue in  https://github.com/containers/qm/issues/164

This PR fix the creating multi images for many nodes,
Now all node container should use node image.

### CODE REVIEW

File: tests/e2e/lib/container [hunk 1]
```diff
@@ -35,20 +35,27 @@ create_container() {
...
+    container_image_name="${4:-base-test-image}"
 
     # Execute build with tag for latest
-    eval "$(podman build \
-            -q \
-            -f "${container_file}" \
-	    --tag "${container_tag}" 1> /dev/null
-    )"
-    if_error_exit "create_container: \
+    image_name=$(podman images \
```
Comment (Yarboa) [inline tests/e2e/lib/container:41]: Please recheck, Does it work without this?

File: tests/e2e/lib/container [hunk 2]
```diff
@@ -35,20 +35,27 @@ create_container() {
...
+            "${container_image_name}" -n
+    )
```
Comment (Yarboa) [inline tests/e2e/lib/container:43]: Does it needed? 
Please recheck main

**Review by Yarboa — COMMENTED** [review]
Please add check if images alredy exist, to avoid trying to recreate images

**Comment by lsm5** [comment]
packit blocked on https://github.com/fedora-copr/copr/issues/2906 . Might wanna ignore packit builds until then if the PR is known to not interfere with rpm builds.

**Comment by rhatdan** [comment]
LGTM

**Comment by Yarboa** [comment]
> packit blocked on [fedora-copr/copr#2906](https://github.com/fedora-copr/copr/issues/2906) . Might wanna ignore packit builds until then if the PR is known to not interfere with rpm builds.

Seems it is fixed :)

**Pull request merged**